SpatialGDK/Source/SpatialGDK/Public/Schema/ComponentPresence.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/Component.h"
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+#include "Utils/SchemaUtils.h"
+
+#include "Containers/Array.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+
+namespace SpatialGDK
+{
+
+struct ComponentPresence : Component
+{
+	static const Worker_ComponentId ComponentId = SpatialConstants::COMPONENT_PRESENCE_COMPONENT_ID;
+
+	ComponentPresence() = default;
+
+	ComponentPresence(TArray<Worker_ComponentId> InActorComponentList)
```

Comment:
Can we add & and move this array?

Reply:
The InActorComponentList should be moved from on the line below, although unless we are going to do 2 overloads I'd argue that by value is better.
(Although 2 overloads also seems good)